src/common/storage/util.js

```diff
@@ -149,6 +150,24 @@ define([
         return deferred.promise;
     }
 
+    /**
+     * Patches a data object to the currently used version. The source version is stored inside the dataObject,
+     * and the target version is coming from the storage constants. If no patch found,
+     * nothing will happen to the object.
+     *
+     * @param {Object} dataObject
+     */
+    function patchDataObject(dataObject) {
+        var dataVersion = dataObject.__v || '0.0.0',
+            myVersion = CONSTANTS.VERSION || '0.0.0';
+
+        if (dataPatches[dataVersion] && typeof dataPatches[dataVersion][myVersion] === 'function') {
+            dataPatches[dataVersion][myVersion](dataObject);
```

Comment:
This is why it should be renamed. Looking here it seems that `dataPatches` are some patch objects for the data...